resources/js/constants/bugConstants.js

```diff
@@ -6,7 +6,7 @@ export const BUG_SEVERITY = [
     {text: 'Critical (Site-breaking)', value: 5}];
 
 export const BUG_TYPES = [
-    {text: 'Design', value: 1}, 
-    {text: 'Language', value: 2},
-    {text: 'Functionality', value: 3}, 
-    {text: 'Other', value: 4}];
\ No newline at end of file
+    {text: 'Design', value: 'Design'}, 
```

Comment:
Make the value all uppercase, I try to keep all constants in uppercase to prevent any case inconsistencies. and also to show it's a constant